méli-mélo/2021-05-conjunction/edge-cases.html

```diff
@@ -16,8 +16,8 @@ <h2 class="h5">Table of content</h2>
 		<li>Edge cases of And/Or decision points (Conjunctions)</li>
 	</ul>
 </nav>
-
-<h2>No border shall not work for responsive break point</h2>
+<p><span class="label label-info">Updated:</span> The design has been updated to allow brdr-0 on all resolutions and will be removed when they become stacked.</p>
+<s><h2>No border shall not work for responsive break point</h2>
```

Comment:
The `<s>` element should be use in the phrasing context not as a flow content.

I suggest to instead put this content inside a details/summary with a title that inform the content are not long accurate. Or simply you can remove those outdated instruction.

What would you like to do?
* [ ] Replicate the `<s>` element for every phrasing context in the deprecated instruction block
* [ ] Move the content inside a details summary with a title saying it is deprecated instruction
* [ ] Remove the outdated instruction.

Reply:
I replicated the `<s>` on every sub element to ensure it is used as phrasing content.